changelog.md

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+0.0.1 (2019-??-??)
+------------------
+
+* Typescript target is now es2019 instead of esnext to ensure that older Node.js versions are supported.
+
+
```

Comment:
Remove this change. The changelog for this package is just meant as a template, not a real list of changes for this package.